hedera/templates/lemmatized_text/create.html

```diff
@@ -19,6 +19,7 @@ <h1>New Text to Lemmatize</h1>
     <div class="form-group">
       <label>Language</label>
       <select name="lang" class="form-control">
+        <option value="zho" {% if select_lang == "zho" %}selected{% endif %}>Chinese</option>
         <option value="grc" {% if select_lang == "grc" %}selected{% endif %}>Greek</option>
         <option value="lat" {% if select_lang == "lat" %}selected{% endif %}>Latin</option>
         <option value="rus" {% if select_lang == "rus" %}selected{% endif %}>Russian</option>
```

Comment:
Not part of the scope of this PR, but this should be refactored to loop over the supported languages defined in settings.